content/zh/docs/concepts/architecture/nodes.md

```diff
@@ -388,188 +526,108 @@ completely unhealthy (i.e. there are no healthy nodes in the cluster). In such
-[NodeRestriction admission plugin](/docs/reference/access-authn-authz/admission-controllers/#noderestriction) are enabled,
-kubelets are only authorized to create/modify their own Node resource.
--->
-启用[节点授权模式](/docs/reference/access-authn-authz/node/) 和
-[NodeRestriction 准入插件](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)时，
-仅授权 kubelet 创建或修改其自己的节点资源。
-
-<!--
-#### Manual Node Administration
-A cluster administrator can create and modify node objects.
--->
-#### 手动节点管理
-
-集群管理员可以创建及修改节点对象。
-
-<!--
-If the administrator wishes to create node objects manually, set the kubelet flag `-register-node=false`.
--->
-如果管理员希望手动创建节点对象，请设置 kubelet 标志 `--register-node=false`。
-
-<!--
-The administrator can modify node resources (regardless of the setting of `-register-node`).
-Modifications include setting labels on the node and marking it unschedulable.
--->
-管理员可以修改节点资源（忽略 `--register-node` 设置）。修改包括在节点上设置标签及
-标记它为不可调度。
-
-<!--
-Labels on nodes can be used in conjunction with node selectors on pods to control scheduling,
-e.g. to constrain a pod to only be eligible to run on a subset of the nodes.
--->
-节点上的标签可以和 Pods 的节点选择算符一起使用来控制调度，例如限制某 pod 只能在符合要求的节点子集上运行。
-
-<!--
-Marking a node as unschedulable prevents new pods from being scheduled to that
-node, but does not affect any existing pods on the node. This is useful as a
-preparatory step before a node reboot, etc. For example, to mark a node
-unschedulable, run this command:
+The Node Controller is also responsible for evicting pods running on nodes with
+`NoExecute` taints, unless the pods do not tolerate the taints.
+The Node Controller also adds {{< glossary_tooltip text="taints" term_id="taint" >}}
+corresponding to node problems like node unreachable or not ready. This means
+that the scheduler won't place Pods onto unhealthy nodes.
 -->
-如果标记节点为不可调度的（unschedulable），将阻止新 Pods 调度到该节点之上，但不会影响任何已经在其上的 Pods。
-这是重启节点等操作之前的一个有用的准备步骤。例如，要标记一个节点为不可调度的，执行以下命令：
-
-```shell
-kubectl cordon $NODENAME
-```
+节点控制器还负责驱逐运行在拥有 `NoExecute` 污点的节点上的 Pod，
+除非这些 Pod 能够容忍此污点。
+节点控制器还负责根据节点故障（例如节点不可访问或没有就绪）为其添加
+{{< glossary_tooltip text="污点" term_id="taint" >}}。
+这意味着调度器不会将 Pod 调度到不健康的节点上。
 
 <!--
-{{< note >}}
-Pods created by a DaemonSet controller bypass the Kubernetes scheduler
-and do not respect the unschedulable attribute on a node. This assumes that daemons belong on
-the machine even if it is being drained of applications while it prepares for a reboot
-{{< /note >}}
+`kubectl cordon` marks a node as 'unschedulable', which has the side effect of the service
+controller removing the node from any LoadBalancer node target lists it was previously 
+eligible for, effectively removing incoming load balancer traffic from the cordoned node(s).
 -->
-
-{{< note >}}
-请注意，被 DaemonSet 控制器创建的 Pods 将忽略 Kubernetes 调度器，且不会遵照节点上不可调度的属性。
-这个假设基于守护程序属于节点机器，即使在准备重启而隔离应用的时候。
-{{< /note >}}
+{{< caution>}}
+`kubectl cordon` 会将节点标记为“不可调度（Unschedulable）”。
```

Comment:
```suggestion
`kubectl cordon` 会将节点标记为“不可调度（Unschedulable）。
```
?

Reply:
no. The Chinese punctuation for quotes appear in pairs too.